crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -479,7 +553,13 @@ where
         });
 
         // Use the vertex shader from the material if present
-        let vert_shader_handle = if let Some(handle) = &self.material_vertex_shader {
+        let vert_shader_handle = if key.mesh_key.contains(MeshPipelineKey::DEFERRED_PREPASS) {
+            if let Some(handle) = &self.deferred_material_vertex_shader {
+                handle.clone()
+            } else {
+                PREPASS_SHADER_HANDLE
```

Comment:
Same here - is this right?

Reply:
The default prepass vertex shader supports deferred just fine.

Reply:
I was mostly asking as it was difficult to keep everything in my head about what was where.